[python-updates] 2024-04-18 - minor updates

mweinelt opened the pull request:
This is a bulk update of python packages, that is restricted to minor version bumps.

### Known issues
- First packages requiring `numpy>=2.0rc1`, workaround in https://github.com/NixOS/nixpkgs/pull/305152/commits/a00fc32b31cd002d8262fe4b15ddabd3e30580da

### Hydra jobset
https://hydra.lossy.network/jobset/nixpkgs/python-updates

### Eval reports
https://shells.darmstadt.ccc.de/~hexa/219.html

mweinelt committed:
python311Packages.dbus-python: 1.2.18 -> 1.3.2

Migrates the build from autotools to meson-python.

Co-Authored-By: adisbladis <adisbladis@gmail.com>
Co-Authored-By: Sandro Jäckel <sandro.jaeckel@gmail.com>

fabaff committed:
python312Packages.types-python-dateutil: 2.8.19.20240106 -> 2.8.19.20240311

fabaff committed:
python311Packages.aiosmtpd: refactor

fabaff committed:
python311Packages.aiosmtpd: add changelog to meta

SuperSandro2000 committed:
python311Packages.aiohttp: remove reference to dev dependencies

dotlambda committed:
python311Packages.pikepdf: 8.13.0 -> 8.14.0

Diff: https://github.com/pikepdf/pikepdf/compare/v8.13.0...v8.14.0

Changelog: https://github.com/pikepdf/pikepdf/blob/v8.14.0/docs/releasenotes/version8.rst

dotlambda committed:
python311Packages.uvicorn: 0.27.1 -> 0.29.0

Diff: https://github.com/encode/uvicorn/compare/refs/tags/0.27.1...0.29.0

Changelog: https://github.com/encode/uvicorn/blob/refs/tags/0.29.0/CHANGELOG.md

mweinelt committed:
python312Packages.markdown: 3.5.2 -> 3.6.0

https://github.com/Python-Markdown/markdown/releases/tag/3.6

fabaff committed:
python312Packages.s3transfer: 0.10.0 -> 0.10.1

Diff: https://github.com/boto/s3transfer/compare/refs/tags/0.10.0...0.10.1

Changelog: https://github.com/boto/s3transfer/blob/0.10.1/CHANGELOG.rst

risicle committed:
python311Packages.httpcore: 1.0.4 -> 1.0.5

risicle committed:
python311Packages.jpylyzer: 2.2.0 -> 2.2.1

update testFiles

risicle committed:
python311Packages.httpcore: add respx to passthru.tests

SuperSandro2000 committed:
ffado: remove unused kernel argument, cleanup python env creation

marsam committed:
python311Packages.orjson: 3.9.15 -> 3.10.1

Diff: https://github.com/ijl/orjson/compare/refs/tags/3.9.15...3.10.1

Changelog: https://github.com/ijl/orjson/blob/3.10.1/CHANGELOG.md

mweinelt committed:
python312Packages.build: 1.1.1 -> 1.2.1

https://github.com/pypa/build/blob/1.2.1/CHANGELOG.rst

mweinelt committed:
python312Packages.setuptools: 69.2.0 -> 69.5.1

https://github.com/pypa/setuptools/blob/v69.5.1/NEWS.rst

mweinelt committed:
python312Packages.pdm-backend: 2.1.8 -> 2.2.1

https://github.com/pdm-project/pdm-backend/releases/tag/2.2.1

fabaff committed:
python312Packages.pyelftools: 0.30 -> 0.31

Diff: https://github.com/eliben/pyelftools/compare/v0.30...v0.31

Changelog: https://github.com/eliben/pyelftools/blob/v0.31/CHANGES

fabaff committed:
python312Packages.pyelftools: refactor

mweinelt committed:
python312Packages.hypothesis: 6.99.12 -> 6.100.1

https://hypothesis.readthedocs.io/en/latest/changes.html#v6-100-1

fabaff committed:
python312Packages.validators: 0.22.0 -> 0.28.0

Diff: https://github.com/python-validators/validators/compare/refs/tags/0.22.0...0.28.0

Changelog: https://github.com/python-validators/validators/blob/0.28.0/CHANGES.md

fabaff committed:
python312Packages.aiodns: 3.1.1 -> 3.2.0

Diff: https://github.com/saghul/aiodns/compare/refs/tags/v3.1.1...v3.2.0

Changelog: https://github.com/saghul/aiodns/releases/tag/v3.2.0

fabaff committed:
python312Packages.referencing: 0.33.0 -> 0.34.0

Diff: https://github.com/python-jsonschema/referencing/compare/refs/tags/v0.33.0...v0.34.0

Changelog: https://github.com/python-jsonschema/referencing/blob/0.34.0/CHANGELOG.rst

mweinelt committed:
python3Packages.accelerate: 0.27.0 -> 0.29.3

https://github.com/huggingface/accelerate/releases/tag/v0.29.3

mweinelt committed:
python3Packages.aioairzone-cloud: 0.4.7 -> 0.5.1

https://github.com/Noltari/aioairzone-cloud/releases/tag/0.5.1

mweinelt committed:
python3Packages.aiorpcx: 0.22.1 -> 0.23.1

mweinelt committed:
python3Packages.aiobotocore: 2.12.1 -> 2.12.3

https://github.com/aio-libs/aiobotocore/releases/tag/2.12.3

mweinelt committed:
python3Packages.albumentations: 1.4.2 -> 1.4.4

https://github.com/albumentations-team/albumentations/releases/tag/1.4.4

mweinelt committed:
python3Packages.allure-behave: 2.13.2 -> 2.13.5

mweinelt committed:
python3Packages.allure-pytest: 2.13.2 -> 2.13.5

mweinelt committed:
python3Packages.allure-python-commons: 2.13.2 -> 2.13.5

mweinelt committed:
python3Packages.allure-python-commons-test: 2.13.4 -> 2.13.5

mweinelt committed:
python3Packages.astropy-healpix: 1.0.2 -> 1.0.3

mweinelt committed:
python3Packages.atpublic: 4.0 -> 4.1.0

mweinelt committed:
python3Packages.awscrt: 0.20.6 -> 0.20.9

https://github.com/awslabs/aws-crt-python/releases/tag/v0.20.9

mweinelt committed:
python3Packages.astroquery: 0.4.6 -> 0.4.7

mweinelt committed:
python3Packages.bdffont: 0.0.17 -> 0.0.20

mweinelt committed:
python3Packages.azure-identity: 1.15.0 -> 1.16.0

https://github.com/Azure/azure-sdk-for-python/blob/azure-identity_1.16.0/sdk/identity/azure-identity/CHANGELOG.md

mweinelt committed:
python3Packages.azure-mgmt-cdn: 13.0.0 -> 13.1.0

https://github.com/Azure/azure-sdk-for-python/blob/azure-mgmt-cdn_13.1.0/sdk/cdn/azure-mgmt-cdn/CHANGELOG.md

mweinelt committed:
python3Packages.beartype: 0.17.2 -> 0.18.3

https://github.com/beartype/beartype/releases/tag/v0.18.3

mweinelt committed:
python3Packages.bids-validator: 1.14.4 -> 1.14.5

https://github.com/bids-standard/bids-validator/releases/tag/v1.14.5

mweinelt committed:
python3Packages.bokeh: 3.3.4 -> 3.4.1

mweinelt committed:
python3Packages.black: 24.3.0 -> 24.4.0

https://github.com/psf/black/blob/24.4.0/CHANGES.md

mweinelt committed:
python3Packages.boxx: 0.10.13 -> 0.10.14

mweinelt committed:
python3Packages.botocore: 1.34.58 -> 1.34.87

https://github.com/boto/botocore/blob/1.34.87/CHANGELOG.rst

mweinelt committed:
python3Packages.casa-formats-io: 0.2.2 -> 0.3.0

mweinelt committed:
python3Packages.cartopy: 0.22.0 -> 0.23.0

mweinelt committed:
python3Packages.cbor2: 5.6.2 -> 5.6.3

https://github.com/agronholm/cbor2/releases/tag/5.6.3

mweinelt committed:
python3Packages.celery: 5.3.6 -> 5.4.0

https://github.com/celery/celery/releases/tag/v5.4.0

mweinelt committed:
python3Packages.cohere: 4.56 -> 4.57

https://github.com/cohere-ai/cohere-python/blob/main/CHANGELOG.md#457

mweinelt committed:
python3Packages.click-didyoumean: 0.3.0 -> 0.3.1

mweinelt committed:
python3Packages.cython: 3.0.9 -> 3.0.10

https://github.com/cython/cython/blob/3.0.10/CHANGES.rst

mweinelt committed:
python3Packages.cvxpy: 1.4.2 -> 1.4.3

https://github.com/cvxpy/cvxpy/releases/tag/v1.4.3

mweinelt committed:
python3Packages.coverage: 7.4.3 -> 7.4.4

mweinelt committed:
python3Packages.dbutils: 3.0.3 -> 3.1.0

mweinelt committed:
python3Packages.databricks-connect: 11.3.26 -> 11.3.33

mweinelt committed:
python3Packages.django-mailman3: 1.3.11 -> 1.3.12

mweinelt committed:
python3Packages.django-configurations: 2.5 -> 2.5.1

mweinelt committed:
python3Packages.django-auth-ldap: 4.7.0 -> 4.8.0

mweinelt committed:
python3Packages.dkimpy: 1.1.5 -> 1.1.6

mweinelt committed:
python3Packages.docplex: 2.25.236 -> 2.27.239

mweinelt committed:
python3Packages.dwdwfsapi: 1.0.7 -> 1.1.0

https://github.com/stephan192/dwdwfsapi/blob/v1.1.0/CHANGELOG.md

mweinelt committed:
python3Packages.ecdsa: 0.18.0 -> 0.19.0

mweinelt committed:
python3Packages.docutils: 0.20.1 -> 0.21.1

mweinelt committed:
python3Packages.execnet: 2.0.2 -> 2.1.1

https://github.com/pytest-dev/execnet/blob/v2.1.1/CHANGELOG.rst

mweinelt committed:
python3Packages.elasticsearch8: 8.12.1 -> 8.13.0

https://github.com/elastic/elasticsearch-py/releases/tag/v8.13.0

mweinelt committed:
python3Packages.elasticsearch: 8.12.1 -> 8.13.0

https://github.com/elastic/elasticsearch-py/releases/tag/v8.13.0

mweinelt committed:
python3Packages.filelock: 3.13.1 -> 3.13.4

https://github.com/tox-dev/py-filelock/releases/tag/3.13.4

mweinelt committed:
python3Packages.faker: 24.0.0 -> 24.11.0

mweinelt committed:
python3Packages.flexmock: 0.12.0 -> 0.12.1

mweinelt committed:
python3Packages.flask: 3.0.2 -> 3.0.3

https://flask.palletsprojects.com/en/3.0.x/changes/#version-3-0-3

mweinelt committed:
python3Packages.fontbakery: 0.11.2 -> 0.12.2

mweinelt committed:
python3Packages.fontmake: 3.8.1 -> 3.9.0

mweinelt committed:
python3Packages.gamble: 0.11 -> 0.13

https://github.com/jpetrucciani/gamble/releases/tag/0.13

mweinelt committed:
python3Packages.geventhttpclient: 2.0.12 -> 2.3.0

https://github.com/geventhttpclient/geventhttpclient/releases/tag/2.3.0

mweinelt committed:
python3Packages.google-api-python-client: 2.125.0 -> 2.126.0

https://github.com/googleapis/google-api-python-client/releases/tag/v2.126.0

mweinelt committed:
python3Packages.glyphsets: 0.6.14 -> 0.6.19

mweinelt committed:
python3Packages.google-auth: 2.28.1 -> 2.29.0

https://github.com/googleapis/google-auth-library-python/blob/v2.29.0/CHANGELOG.md

mweinelt committed:
python3Packages.google-cloud-iam: 2.14.3 -> 2.15.0

https://github.com/googleapis/python-iam/releases/tag/v2.15.0

mweinelt committed:
python3Packages.google-cloud-monitoring: 2.19.3 -> 2.21.0

https://github.com/googleapis/google-cloud-python/tree/google-cloud-monitoring-v2.21.0/packages/google-cloud-monitoring

mweinelt committed:
python3Packages.google-cloud-kms: 2.21.3 -> 2.21.4

https://github.com/googleapis/google-cloud-python/blob/google-cloud-kms-v2.21.4/packages/google-cloud-kms/CHANGELOG.md

mweinelt committed:
python3Packages.gradio: 4.22.0 -> 4.27.0

mweinelt committed:
python3Packages.google-cloud-spanner: 3.44.0 -> 3.45.0

https://github.com/googleapis/python-spanner/blob/v3.45.0/CHANGELOG.md

mweinelt committed:
python3Packages.grpcio-channelz: 1.62.1 -> 1.62.2

mweinelt committed:
python3Packages.grpcio: 1.62.1 -> 1.62.2

mweinelt committed:
python3Packages.grpcio-testing: 1.62.1 -> 1.62.2

mweinelt committed:
python3Packages.grpcio-status: 1.62.1 -> 1.62.2

mweinelt committed:
python3Packages.h5py: 3.10.0 -> 3.11.0

https://github.com/h5py/h5py/blob/3.11.0/docs/whatsnew/3.11.rst

mweinelt committed:
python3Packages.grpcio-tools: 1.62.1 -> 1.62.2

mweinelt committed:
python3Packages.ibm-cloud-sdk-core: 3.19.2 -> 3.20.0

https://github.com/IBM/python-sdk-core/blob/v3.20.0/CHANGELOG.md

mweinelt committed:
python3Packages.hatchling: 1.22.4 -> 1.24.1

https://github.com/pypa/hatch/releases/tag/hatchling-v1.24.1

mweinelt committed:
python3Packages.importlib-metadata: 7.0.2 -> 7.1.0

mweinelt committed:
python3Packages.ipympl: 0.9.3 -> 0.9.4

mweinelt committed:
python3Packages.importlib-resources: 6.1.3 -> 6.3.2

mweinelt committed:
python3Packages.inflect: 7.0.0 -> 7.2.0

https://github.com/jaraco/inflect/blob/v7.2.0/CHANGES.rst

mweinelt committed:
python3Packages.itsdangerous: 2.1.2 -> 2.2.0

https://github.com/pallets/itsdangerous/blob/2.2.0/CHANGES.rst

mweinelt committed:
python3Packages.jaraco-collections: 5.0.0 -> 5.0.1

https://github.com/jaraco/jaraco.collections/blob/v5.0.1/NEWS.rst

mweinelt committed:
python3Packages.joblib: 1.3.2 -> 1.4.0

https://github.com/joblib/joblib/releases/tag/1.4.0

mweinelt committed:
python3Packages.jaraco-functools: 4.0.0 -> 4.0.1

mweinelt committed:
python3Packages.jsonpickle: 3.0.3 -> 3.0.4

mweinelt committed:
python3Packages.keyrings-alt: 5.0.0 -> 5.0.1

https://github.com/jaraco/keyrings.alt/blob/v5.0.1/NEWS.rst

mweinelt committed:
python3Packages.keras: 3.0.5 -> 3.2.1

mweinelt committed:
python3Packages.lmfit: 1.2.2 -> 1.3.0

mweinelt committed:
python3Packages.ledgerblue: 0.1.48 -> 0.1.50

mweinelt committed:
python3Packages.kombu: 5.3.5 -> 5.3.7

https://github.com/celery/kombu/blob/v5.3.7/Changelog.rst

mweinelt committed:
python3Packages.magic-wormhole: 0.13.0 -> 0.14.0

https://github.com/magic-wormhole/magic-wormhole/blob/0.14.0/NEWS.md

mweinelt committed:
python3Packages.localstack-ext: 3.2.0 -> 3.3.0

mweinelt committed:
python3Packages.magika: 0.5.0 -> 0.5.1

mweinelt committed:
python3Packages.mako: 1.3.2 -> 1.3.3

https://docs.makotemplates.org/en/latest/changelog.html

mweinelt committed:
python3Packages.matplotlib: 3.8.3 -> 3.8.4

https://github.com/matplotlib/matplotlib/releases/tag/v3.8.4

mweinelt committed:
python3Packages.maxminddb: 2.5.2 -> 2.6.1

https://github.com/maxmind/MaxMind-DB-Reader-python/blob/v2.6.1/HISTORY.rst

mweinelt committed:
python3Packages.mecab-python3: 1.0.8 -> 1.0.9

https://github.com/SamuraiT/mecab-python3/releases/tag/v1.0.9

mweinelt committed:
python3Packages.matplotlib-inline: 0.1.6 -> 0.1.7

mweinelt committed:
python3Packages.meson-python: 0.15.0 -> 0.16.0

https://github.com/mesonbuild/meson-python/blob/0.16.0/CHANGELOG.rst

mweinelt committed:
python3Packages.mlflow: 2.11.3 -> 2.12.1

https://github.com/mlflow/mlflow/blob/v2.12.1/CHANGELOG.md

mweinelt committed:
python3Packages.mocket: 3.12.4 -> 3.12.5

https://github.com/mindflayer/python-mocket/releases/tag/3.12.5

mweinelt committed:
python3Packages.mypy-protobuf: 3.5.0 -> 3.6.0

mweinelt committed:
python3Packages.mpi4py: 3.1.5 -> 3.1.6

mweinelt committed:
python3Packages.netcdf4: 1.6.2 -> 1.6.5

https://github.com/Unidata/netcdf4-python/raw/v1.6.5/Changelog

mweinelt committed:
python3Packages.nilearn: 0.10.3 -> 0.10.4

https://github.com/nilearn/nilearn/releases/tag/0.10.4

mweinelt committed:
python3Packages.networkx: 3.2.1 -> 3.3

https://github.com/networkx/networkx/blob/networkx-3.3/doc/release/release_3.3.rst

mweinelt committed:
python3Packages.numba-scipy: 0.3.1 -> 0.4.0

https://github.com/numba/numba-scipy/blob/master/CHANGE_LOG

mweinelt committed:
python3Packages.numexpr: 2.9.0 -> 2.10.0

mweinelt committed:
python3Packages.panel: 1.3.8 -> 1.4.1

https://github.com/holoviz/panel/releases/tag/v1.4.1

mweinelt committed:
python3Packages.paddle2onnx: 1.1.0 -> 1.2.0

https://github.com/PaddlePaddle/Paddle2ONNX/releases/tag/v1.2.0

mweinelt committed:
python3Packages.parsedmarc: 8.8.0 -> 8.11.0

https://github.com/domainaware/parsedmarc/blob/master/CHANGELOG.md#8110

mweinelt committed:
python3Packages.parsel: 1.9.0 -> 1.9.1

mweinelt committed:
python3Packages.parso: 0.8.3 -> 0.8.4

https://github.com/davidhalter/parso/blob/master/CHANGELOG.rst

mweinelt committed:
python3Packages.path: 16.10.0 -> 16.14.0

https://github.com/jaraco/path/blob/v16.14.0/NEWS.rst

mweinelt committed:
python3Packages.pg8000: 1.30.5 -> 1.31.1

https://github.com/tlocke/pg8000#release-notes

mweinelt committed:
python3Packages.pathlib-abc: 0.1.1 -> 0.2.0

https://github.com/barneygale/pathlib-abc/blob/0.2.0/CHANGES.rst

mweinelt committed:
python3Packages.phonenumbers: 8.13.31 -> 8.13.34

https://github.com/daviddrysdale/python-phonenumbers/blob/v8.13.34/python/HISTORY.md

mweinelt committed:
python3Packages.pixel-font-builder: 0.0.15 -> 0.0.19

mweinelt committed:
python3Packages.pycparser: 2.21 -> 2.22

mweinelt committed:
python3Packages.plotly: 5.20.0 -> 5.21.0

mweinelt committed:
python3Packages.pyasn1: 0.5.1 -> 0.6.0

https://github.com/etingof/pyasn1/blob/master/CHANGES.rst

mweinelt committed:
python3Packages.pycups: 2.0.1 -> 2.0.4

mweinelt committed:
python3Packages.pyerfa: 2.0.1.1 -> 2.0.1.4

mweinelt committed:
python3Packages.pyfakefs: 5.3.5 -> 5.4.1

https://github.com/jmcgeheeiv/pyfakefs/blob/v5.4.1/CHANGES.md

mweinelt committed:
python3Packages.pykalman: 0.9.5 -> 0.9.7

mweinelt committed:
python3Packages.pykdtree: 1.3.11 -> 1.3.12

mweinelt committed:
python3Packages.pymysql: 1.0.2 -> 1.1.0

mweinelt committed:
python3Packages.pyopenssl: 24.0.0 -> 24.1.0

https://github.com/pyca/pyopenssl/blob/24.1.0/CHANGELOG.rst

mweinelt committed:
python3Packages.pymongo: 4.6.2 -> 4.6.3

mweinelt committed:
python3Packages.pyproject-metadata: 0.7.1 -> 0.8.0

https://github.com/FFY00/python-pyproject-metadata/blob/0.8.0/CHANGELOG.rst

mweinelt committed:
python3Packages.pyqt-builder: 1.15.4 -> 1.16.0

mweinelt committed:
python3Packages.pysimplegui: 5.0.3 -> 5.0.4

mweinelt committed:
python3Packages.pyramid-beaker: 0.8 -> 0.9

mweinelt committed:
python3Packages.pytest-mock: 3.12.0 -> 3.14.0

https://github.com/pytest-dev/pytest-mock/blob/v3.14.0/CHANGELOG.rst

mweinelt committed:
python3Packages.python-musicpd: 0.8.0 -> 0.9.0

mweinelt committed:
python3Packages.pywavelets: 1.4.1 -> 1.6.0

mweinelt committed:
python3Packages.qdldl: 0.1.7.post1 -> 0.1.7.post2

mweinelt committed:
python3Packages.pywbem: 1.6.3 -> 1.7.1

https://github.com/pywbem/pywbem/blob/1.7.1/docs/changes.rst

mweinelt committed:
python3Packages.reproject: 0.13.0 -> 0.13.1

https://github.com/astropy/reproject/releases/tag/v0.13.1

mweinelt committed:
python3Packages.reportlab: 4.1.0 -> 4.2.0

mweinelt committed:
python3Packages.redis: 5.0.1 -> 5.0.3

https://github.com/redis/redis-py/releases/tag/v5.0.3

mweinelt committed:
python3Packages.requests-mock: 1.11.0 -> 1.12.1

https://github.com/jamielennox/requests-mock/releases/tag/1.12.1

mweinelt committed:
python3Packages.rpy2: 3.5.15 -> 3.5.16

mweinelt committed:
python3Packages.scalene: 1.5.38 -> 1.5.39

https://github.com/plasma-umass/scalene/releases/tag/v1.5.39

mweinelt committed:
python3Packages.safety: 3.0.1 -> 3.1.0

https://github.com/pyupio/safety/blob/3.1.0/CHANGELOG.md

mweinelt committed:
python3Packages.schwifty: 2024.1.1.post0 -> 2024.4.0

https://github.com/mdomke/schwifty/blob/2024.4.0/CHANGELOG.rst

mweinelt committed:
python3Packages.scikit-bio: 0.5.9 -> 0.6.0

mweinelt committed:
python3Packages.scikit-learn: 1.4.1.post1 -> 1.4.2

https://scikit-learn.org/stable/whats_new/v1.4.html#version-1-4-2

mweinelt committed:
python3Packages.snowflake-sqlalchemy: 1.5.1 -> 1.5.3

https://github.com/snowflakedb/snowflake-sqlalchemy/blob/v1.5.3/DESCRIPTION.md

mweinelt committed:
python3Packages.slicer: 0.0.7 -> 0.0.8

mweinelt committed:
python3Packages.sigstore-protobuf-specs: 0.3.0 -> 0.3.1

mweinelt committed:
python3Packages.spglib: 2.3.1 -> 2.4.0

https://github.com/spglib/spglib/raw/v2.4.0/ChangeLog

mweinelt committed:
python3Packages.sphinx-autodoc-typehints: 2.0.0 -> 2.1.0

https://github.com/tox-dev/sphinx-autodoc-typehints/releases/tag/2.1.0

mweinelt committed:
python3Packages.sphinx-autobuild: 2024.2.4 -> 2024.4.16

mweinelt committed:
python3Packages.sphinxcontrib-tikz: 0.4.18 -> 0.4.19

mweinelt committed:
python3Packages.spyder: 5.5.3 -> 5.5.4

https://github.com/spyder-ide/spyder/blob/master/CHANGELOG.md

mweinelt committed:
python3Packages.sqlparse: 0.4.4 -> 0.5.0

mweinelt committed:
python3Packages.sqlalchemy-utils: 0.41.1 -> 0.41.2

https://github.com/kvesteri/sqlalchemy-utils/releases/tag/0.41.2

mweinelt committed:
python3Packages.statsmodels: 0.14.1 -> 0.14.2

https://github.com/statsmodels/statsmodels/releases/tag/v0.14.2

mweinelt committed:
python3Packages.stripe: 8.9.0 -> 8.11.0

https://github.com/stripe/stripe-python/blob/v8.11.0/CHANGELOG.md

mweinelt committed:
python3Packages.sunpy: 5.1.1 -> 5.1.2

mweinelt committed:
python3Packages.tablib: 3.5.0 -> 3.6.1

https://github.com/jazzband/tablib/raw/v3.6.1/HISTORY.md

mweinelt committed:
python3Packages.tifffile: 2024.2.12 -> 2024.4.18

https://github.com/cgohlke/tifffile/blob/v2024.4.18/CHANGES.rst

mweinelt committed:
python3Packages.tidalapi: 0.7.5 -> 0.7.6

https://github.com/tamland/python-tidal/releases/tag/v0.7.6

mweinelt committed:
python3Packages.traitlets: 5.14.1 -> 5.14.2

https://github.com/ipython/traitlets/blob/v5.14.2/CHANGELOG.md

mweinelt committed:
python3Packages.trio: 0.24.0 -> 0.25.0

mweinelt committed:
python3Packages.trytond: 7.0.9 -> 7.0.10

https://foss.heptapod.net/tryton/tryton/-/blob/trytond-7.0.10/trytond/CHANGELOG?ref_type=tags

mweinelt committed:
python3Packages.trove-classifiers: 2024.3.3 -> 2024.4.10

https://github.com/pypa/trove-classifiers/releases/tag/2024.4.10

mweinelt committed:
python3Packages.typed-settings: 24.1.0 -> 24.2.0

https://gitlab.com/sscherfke/typed-settings/-/blob/24.2.0/CHANGELOG.rst

mweinelt committed:
python3Packages.type-infer: 0.0.18 -> 0.0.19

mweinelt committed:
python3Packages.typer: 0.9.0 -> 0.12.3

https://github.com/tiangolo/typer/releases/tag/0.12.3

mweinelt committed:
python3Packages.types-protobuf: 4.24.0.20240311 -> 4.25.0.20240417

mweinelt committed:
python3Packages.types-pillow: 10.2.0.20240331 -> 10.2.0.20240415

mweinelt committed:
python3Packages.types-pyopenssl: 24.0.0.20240311 -> 24.0.0.20240417

mweinelt committed:
python3Packages.types-python-dateutil: 2.8.19.20240311 -> 2.9.0.20240316

mweinelt committed:
python3Packages.types-pytz: 2024.1.0.20240203 -> 2024.1.0.20240417

mweinelt committed:
python3Packages.types-pyyaml: 6.0.12.12 -> 6.0.12.20240311

mweinelt committed:
python3Packages.types-redis: 4.6.0.20240409 -> 4.6.0.20240417

mweinelt committed:
python3Packages.types-setuptools: 69.2.0.20240317 -> 69.5.0.20240415

mweinelt committed:
python3Packages.types-s3transfer: 0.10.0 -> 0.10.1

mweinelt committed:
python3Packages.types-toml: 0.10.8.7 -> 0.10.8.20240310

mweinelt committed:
python3Packages.types-tqdm: 4.66.0.20240106 -> 4.66.0.20240417

mweinelt committed:
python3Packages.typing-extensions: 4.10.0 -> 4.11.0

https://github.com/python/typing_extensions/blob/4.11.0/CHANGELOG.md

mweinelt committed:
python3Packages.ufo2ft: 3.1.0 -> 3.2.1

https://github.com/googlefonts/ufo2ft/releases/tag/v3.2.1

mweinelt committed:
python3Packages.uncompyle6: 3.9.0 -> 3.9.1

mweinelt committed:
python3Packages.uqbar: 0.7.3 -> 0.7.4

https://github.com/josiah-wolf-oberholtzer/uqbar/releases/tag/v0.7.4

mweinelt committed:
python3Packages.virt-firmware: 24.2 -> 24.4

mweinelt committed:
python3Packages.versioningit: 3.0.0 -> 3.1.0

https://versioningit.readthedocs.io/en/latest/changelog.html

mweinelt committed:
python3Packages.virtualenv: 20.25.1 -> 20.25.3

https://github.com/pypa/virtualenv/blob/20.25.3/docs/changelog.rst

mweinelt committed:
python3Packages.werkzeug: 3.0.1 -> 3.0.2

https://werkzeug.palletsprojects.com/en/3.0.x/changes/#version-3-0-2

mweinelt committed:
python3Packages.weaviate-client: 4.5.1 -> 4.5.5

https://github.com/weaviate/weaviate-python-client/releases/tag/v4.5.5

mweinelt committed:
python3Packages.xmlsec: 1.3.13 -> 1.3.14

mweinelt committed:
python3Packages.ydiff: 1.2 -> 1.3

mweinelt committed:
python3Packages.zarr: 2.17.1 -> 2.17.2

https://github.com/zarr-developers/zarr-python/releases/tag/v2.17.2

mweinelt committed:
python3Packages.zephyr-python-api: 0.0.4 -> 0.0.5

mweinelt committed:
python3Packages.zipp: 3.17.0 -> 3.18.1

mweinelt committed:
python311Package.css-inline: 0.13.0 -> 0.14.0

https://github.com/Stranger6667/css-inline/releases/tag/python-v0.14.0

mweinelt committed:
python311Packages.numpy: relax meson-python constraint

mweinelt committed:
python311Packages.sphinx: 7.2.6 -> 7.3.7

mweinelt committed:
python312Packages.sphinxcontrib-jquery: provide defusedxml for tests

mweinelt committed:
python312Packages.scikit-build: disable failing tests

mweinelt committed:
python311Packages.sphinx-pytest: provide defusedxml for tests

mweinelt committed:
python312Packages.sphinx-rtd-theme: disable failing test

fabaff committed:
python312Packages.sqlparse: refactor

fabaff committed:
python312Packages.readme-renderer: clean-up disabled tests

mweinelt commented:
https://shells.darmstadt.ccc.de/~hexa/200.html

mweinelt commented:
https://shells.darmstadt.ccc.de/~hexa/203.html

jonringer committed:
python3Packages.qrcode: disable sandbox-incompatible test

mweinelt commented:
https://shells.darmstadt.ccc.de/~hexa/206.html

jonringer committed:
python3Packages.myst-parser: disable sphinx sensitive tests

mweinelt committed:
python312Packages.trio-websocket: 0.10.2 -> 0.11.1

https://github.com/python-trio/trio-websocket/blob/0.11.1/CHANGELOG.md

And disable failing tests due to trio 0.25.0 upgrade.

mweinelt committed:
python312Packages.zlib-ng: 0.4.1 -> 0.4.3

https://github.com/pycompression/python-zlib-ng/blob/v0.4.3/CHANGELOG.rst

drupol commented:
Can we upgrade to sphinx 7.3.7 in here so I can close https://github.com/NixOS/nixpkgs/pull/305335 ?

mweinelt commented:
https://shells.darmstadt.ccc.de/~hexa/211.html

mweinelt committed:
python312Packages.argcomplete: 3.2.3 -> 3.3.0

https://github.com/kislyuk/argcomplete/blob/v3.3.0/Changes.rst

mweinelt committed:
python312Packages.aiohttp: 3.9.4 -> 3.9.5

https://github.com/aio-libs/aiohttp/blob/v3.9.5/CHANGES.rst

mweinelt committed:
python312Packages.python-jose: disable failing tests

These started failing with ecdas 0.19.0, because representation of ecdsa
keys changed slightly, which broke comparison with the fixtures.

mweinelt committed:
amend! python3Packages.geventhttpclient: 2.0.12 -> 2.3.1

python3Packages.geventhttpclient: 2.0.12 -> 2.3.0

https://github.com/geventhttpclient/geventhttpclient/releases/tag/2.1.0
https://github.com/geventhttpclient/geventhttpclient/releases/tag/2.1.1
https://github.com/geventhttpclient/geventhttpclient/releases/tag/2.2.0
https://github.com/geventhttpclient/geventhttpclient/releases/tag/2.2.1
https://github.com/geventhttpclient/geventhttpclient/releases/tag/2.3.1
https://github.com/geventhttpclient/geventhttpclient/releases/tag/2.3.1

mweinelt committed:
python312Packages.elastic-transport: 8.12.0 -> 8.13.0

https://github.com/elastic/elastic-transport-python/releases/tag/v8.13.0

mweinelt commented:
https://shells.darmstadt.ccc.de/~hexa/213.html

mweinelt committed:
python312Packages.fastapi: 0.110.0 -> 0.110.2

https://github.com/tiangolo/fastapi/releases/tag/0.110.2

mweinelt committed:
python312Packages.django-bootstrap3: 23.6 -> 24.2

https://github.com/zostera/django-bootstrap3/blob/v24.2/CHANGELOG.md

mweinelt committed:
python312Packages.django-bootstrap4: 24.1 -> 24.3

https://github.com/zostera/django-bootstrap4/blob/refs/tags/v24.3/CHANGELOG.md

mweinelt commented:
https://shells.darmstadt.ccc.de/~hexa/216.html

mweinelt committed:
python312Packages.django-bootstrap5: 23.3 -> 24.1

https://github.com/zostera/django-bootstrap5/blob/v24.1/CHANGELOG.md

mweinelt committed:
python312Packages.hypercorn: 0.14.3 -> 0.16.0

https://github.com/pgjones/hypercorn/blob/0.16.0/CHANGELOG.rst

mweinelt commented:
https://shells.darmstadt.ccc.de/~hexa/219.html

mweinelt committed:
python312Packages.dulwich: disable failing test

dotlambda committed:
python311Packages.trio-asyncio: 0.14.0 -> 0.14.1

Diff: https://github.com/python-trio/trio-asyncio/compare/refs/tags/v0.14.0...v0.14.1

Changelog: https://github.com/python-trio/trio-asyncio/blob/v0.14.1/docs/source/history.rst

dotlambda committed:
python311Packages.trio: refactor

mweinelt commented on pkgs/development/python-modules/trio-asyncio/default.nix:
```diff
@@ -43,16 +44,18 @@ buildPythonPackage rec {
-  disabledTestPaths = [
-    "tests/python" # tries to import internal API test.test_asyncio
+  disabledTests = [
+    #"test_trio_asyncio_cancel_direct"
```
leftover?

mweinelt committed:
python312Packages.avro: disable

mweinelt committed:
python312Packages.kazoo: replace nose with pynose

mweinelt committed:
home-assistant: disable versioningit tests

mweinelt committed:
python312Packages.dbt-core: 1.7.12 -> 1.7.13

https://github.com/dbt-labs/dbt-core/blob/v1.7.13/CHANGELOG.md

mweinelt committed:
python312Packages.pymdown-extensions: disable failing test

mweinelt committed:
python312Packages.rq: disable failing test

mweinelt committed:
python312Packages.sphinx-autoapi: disable failing test

mweinelt commented:
@ofborg eval

mweinelt commented:
Eval fails due to #288435.
